library/solr

```diff
@@ -8,32 +8,42 @@ Maintainers: The Apache Solr Project <dev@solr.apache.org> (@asfbot),
 GitRepo: https://github.com/apache/solr-docker.git
 GitFetch: refs/heads/main
 
-Tags: 9.5.0, 9.5, 9, latest
+Tags: 9.6.0, 9.6, 9, latest
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: 47b84007a29fa9a9d47917e1ba7f82cf863a5c93
+GitCommit: e5a988a1d36ac74714ff58117f0d99a9466bd3d1
+Directory: 9.6
+
+Tags: 9.6.0-slim, 9.6-slim, 9-slim, slim
+Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
+GitCommit: e5a988a1d36ac74714ff58117f0d99a9466bd3d1
+Directory: 9.6-slim
+
+Tags: 9.5.0, 9.5
```

Comment:
Apologies if I've asked this before, but are 9.5 and 9.4 still actively supported/updated/patched now that 9.6.0 is out?

(If not, they should be removed here, but if so, all good :heart:)

Reply:
https://solr.apache.org/downloads.html#about-versions-and-support seems to suggest that 9.x and 8.11.x are the two currently supported release lines, corroborating that this PR should be removing 9.5 (and 9.4 :see_no_evil:) :eyes:

(If this is accurate, I'd even suggest going as far as to rename the 9.6 folder in your repository to simply "9" to mirror the actual level of supported lines, but that might require more changes in your automation/scripting/repo management :eyes:)

Reply:
This is correct. We've been here before.

May I suggest that we go on with this release, and then file a followup PR to prune unsupported versions? Also we should improve our release-process to do this automatically...